apps/src/p5lab/AnimationPicker/AnimationPickerListItem.jsx

```diff
@@ -35,13 +36,17 @@ export default class AnimationPickerListItem extends React.Component {
       label,
       selected,
       isBackgroundsTab,
+      isAnimationJsonMode,
     } = this.props;
     const {loaded, hover} = this.state;
 
-    const iconImageSrc = category
+    let iconImageSrc = category
       ? `/blockly/media/p5lab/animation-previews/category_${category}.png`
       : '';
-
+    if (isAnimationJsonMode && category === 'all') {
+      iconImageSrc =
+        '/blockly/media/p5lab/animation-previews/category_all_including_backgrounds.png';
+    }
```

Comment:
The 'category_all.png` image is currently identical to the 'category_level_animations.png' image on the [Generate Animation JSON tool](https://levelbuilder-studio.code.org/sprites/select_start_animations?library=all) page which is confusing to content editors since the 'all' category includes both costumes and backgrounds and the 'level_animations' category includes only costumes.

Reply:
Nit: not your change, but would be great to update this to use a import statement (plays nicely with Storybook):

https://github.com/code-dot-org/code-dot-org/blob/11012fd30f01eae878ae3e57f8007fa7f34e6627/apps/src/code-studio/components/ShareAllowedDialog.jsx#L26

Reply:
Attempted dynamic import, but didn't seem to work: https://github.com/code-dot-org/code-dot-org/pull/55333/commits/06f21372734f779bccb79ffe2236baed20d3f39e